libs/elastic-indexing/src/lib/indexing.service.ts

```diff
@@ -43,4 +46,48 @@ export class IndexingService {
     logger.info('Done with sync')
     return true
   }
+
+  async rankEvaluation(locale: string) {
+    const requests = await this.getRankQueries(locale)
+    const index = await this.getIndex(locale)
```

Comment:
You should be able to get the ES indexes from `SearchIndexes[locale]`

Reply:
The alias points to the current index, is there a need to rank other index versions?